src/index.js

```diff
@@ -2,0 +1,43 @@
+// 'use strict';
+
+// import express from 'express';
+// import cors from 'cors';
```

Comment:
The import statement for CORS is commented out. Uncomment this line to use CORS in your application.